DataCert/retrieveBRILData.py

```diff
@@ -63,6 +72,7 @@ def CallBrilcalcAndProcess(onlineLumiDict,type="best"):
 parser.add_argument('-o', '--overwrite', action='store_true', default=False, help="Overwrite data if it already exists (default False)")
 parser.add_argument('--datadir',    type=str, default="brildata", help="Location to put/retrieve bril data")
 parser.add_argument('-x', '--xing', action='store_true', default=False, help="Get the Lumi per BX")
+parser.add_argument('-n', '--normtag', action='store_true', default=False, help="--normaltag option for brilcalc")
```

Comment:
I want the default to be true.  You can change the comment to "Toggle --normaltag option for brilcalc [Default: True]".